.gitignore

```
@@ -162,5 +162,6 @@ cython_debug/
 
 *-versions.json
 *-index.html
-k3s.json
-rke2.json
\ No newline at end of file
+
+# ignore the data location
+data/
```

technically the k3s.json is created in the root directory, but we can keep both
```suggestion
data/
k3s.json
rke2.json
```